Webinar details styling sidney

Sidopjescherm opened the pull request:
Nu heb ik de gehele webinars detail pagina gemaakt is met focus op mobiel, maar ook responsive is op desktop. 

Ik heb getest op
 
- [x] User test
- [x] Tab test
- [x] Browser test (edge, chrome en firefox. Zou één van jullie kunnen testen op Safari?)
 
## afbeeldinen
 
<img width="1498" height="815" alt="desktop" src="https://github.com/user-attachments/assets/d6b2b134-c4e6-450d-b201-122019c16fa6" />
<img width="991" height="824" alt="mobile" src="https://github.com/user-attachments/assets/fbf54222-820f-4da6-9dca-b99014505ec6" />

 
 
## Om te reviewen:

- Browser test op safari
- Welke code je zou veranderen
- Design dat je zou aanpassen

Sidopjescherm committed:
#270 Aanpassing globale css

Ik heb de elementen die ik gebruik op mijn pagina ook toegevoegd aan de styleguide zodat het meteen zal worden aangepast

Sidopjescherm committed:
#270 Filteren zodat de categorieën er ook zijn

Na het bekijken van het project dat vorige jaar draaide heb ik de fetch aangepast zodat we de velden fetchen die we nodig hebben.

Sidopjescherm committed:
#270 HTML

Na het fetchen van de jusite velden moest ik id opschrijven en [0] zodat je de juiste aanpakt.

Sidopjescherm committed:
#270 stylesheet

Hier heb ik voor gezorgd dat alles op de mobiele apparaten het toegankelijk begint te worden

Sidopjescherm committed:
#301 Layout grid

Wat moet ik hieraan veranderen om het werkend te krijgen @nadiachaja of @KyanTG.

Om te kijken hoe ik het grid heb getekend zie je dit in een comment in issue #270

Sidopjescherm committed:
#270 Groot scherm en klein scherm

nu heb ik de jusite media-queries toegevoegd voor grote schermen en kleine schermen

Sidopjescherm committed:
#270 Volgorde gewijzigd

Ik hen de volgorde gewijzigd zodat het logischer is wanneer je door de pagina heen 'tabbed'

Sidopjescherm committed:
#270 aligning center

nadiachaja reviewed (changes requested):
Ik zou het kruimelpad en de webinars titel ook wat ruimte geven. 
voor het kruimelpad is dit op mobile en desktop en voor de webinars titel is het voor mobile. 
<img width="202" height="81" alt="Scherm­afbeelding 2025-10-20 om 16 22 36" src="https://github.com/user-attachments/assets/0d08e0a2-922d-4c27-933d-fd7d537eb57c" />
<img width="320" height="107" alt="Scherm­afbeelding 2025-10-20 om 16 30 15" src="https://github.com/user-attachments/assets/7ed00b63-9ebe-45e9-bf77-66ac570794da" />


Ik heb in de global css ook de border-radius toegevoegd ik heb in het figma bestand gekeken welke border-radius daar stonden. 
Ik zou  de border-radius uit de global css gebruiken dan is alles hetzelfde.

Ik zou nog een border om de description doen dat is voor de zekerheid als blauwe achtergrond niet te zien is. 

Ik vind zelf dat de blauwe vlak bij de Q&A en de subtitles opvallen en dat de titel en de video wat naar de achtergrond gaan. 
<img width="1440" height="252" alt="Scherm­afbeelding 2025-10-20 om 16 29 57" src="https://github.com/user-attachments/assets/afa9a68e-47b3-42a8-8766-7c7d0828b4f1" />


Waarom heb je ervoor gekozen om de comments niet erin te zetten alleen html en css?

KyanTG commented:
Ik ben het met Nadia eens, voor de rest is het volgensmij allemaal goed! Ik begrijp de keuze van de blauwe balk bij subtitles wel alleen ik denk inderdaad ook dat daardoor de focus daarop valt ipv.  op de video

Sidopjescherm committed:
#304 Aanpassing feedback

Ik heb de feedback die jullie hebben gegeven aangepast. Wat is er veranderd?

- Border om de description heen
- Border radius genomen uit de globale css
- ruimte toegevoegd bij de header en de titel
- Donkerblauw nummer drie gebruikt zodat de subtitle niet meteen opvalt.

@nadiachaja @KyanTG

KyanTG commented:
Ziet er goed uit! ik zie wel dat er een conflict is met de global css nu. Voor de rest mag je hem van mij mergen

nadiachaja commented:
Ik zie iets kleins de Ask the question button heeft op desktop een andere padding dan op mobile de tablet. Ook zou ik nog bij de datum en categorie  align-self:center toevoegen dan staat de tekst in het midden. 
<img width="356" height="289" alt="Scherm­afbeelding 2025-10-21 om 11 45 54" src="https://github.com/user-attachments/assets/7e850bc2-db26-4374-bdfe-a33914f1f49a" />

Verder ziet het er goed uit.

Sidopjescherm committed:
#304 Feedback Nadia verwerkt

@nadiachaja je had idnerdaad groot gelijk met de align-self: center en de padding. Beide de li en de a hebben nu 1em

Sidopjescherm committed:
Merge branch 'dev' into webinar-details-styling-sidney

nadiachaja reviewed (approved):
ziet er goed uit
